gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -353,6 +360,7 @@ class BaselineIdea extends AbstractBaselinePlugin {
     }
 
     private static void addInspectionProjectProfile(node) {
+        // language=xml
```

Comment:
i stumbled across this trick further down this file, turns out IntelliJ can give you language-specific code highlighting for snippets!!!
![image](https://user-images.githubusercontent.com/3473798/99422941-f2c59400-28f7-11eb-9590-0e21da94be2b.png)